projects/kit/components/slider/slider.component.ts

```diff
@@ -34,13 +41,14 @@ export class TuiSliderComponent {
     private readonly control = inject(NgControl, {self: true, optional: true});
 
     protected readonly options = inject(TUI_SLIDER_OPTIONS);
+    protected readonly segments = signal<number[]>([1]);
+    protected readonly ticksGradient = computed((segments = this.segments()) =>
+        this.getTicksGradient(segments),
```

Comment:
can just be `this.getTicksGradient(this.segments())`. We move signals up that way if they are used multiple times or if we want to make sure they are definitely read so that computed depends on them.